qcfractal/interface/collections/reaction_dataset.py

```diff
@@ -11,7 +11,7 @@
 
 from .collection_utils import nCr, register_collection
 from .dataset import Dataset
-from ..dict_utils import replace_dict_keys
+from qcfractal.interface.util import replace_dict_keys
```

Comment:
Why switch this? Becomes different than all other imports.

Reply:
Because the refactor did it. I'll revert it